http.c

```diff
@@ -179,6 +179,8 @@ static int http_schannel_check_revoke_mode =
  */
 static int http_schannel_use_ssl_cainfo;
 
+static int http_schannel_auto_client_cert = 0;
```

Comment:
`static` variables are implicitly initialized, therefore the ` = 0` is unneeded. Let's drop it.

Also, cURL's naming does not specify that this is specific to Secure Channel, so I would also drop the `schannel_` infix.